sql/core/src/main/scala/org/apache/spark/sql/execution/python/AggregateInPandasExec.scala

```diff
@@ -137,13 +135,12 @@ case class AggregateInPandasExec(
 
       val columnarBatchIter = new ArrowPythonRunner(
         pyFuncs,
-        bufferSize,
-        reuseWorker,
         PythonEvalType.SQL_GROUPED_AGG_PANDAS_UDF,
         argOffsets,
         aggInputSchema,
         sessionLocalTimeZone,
-        pythonRunnerConf).compute(projectedRowIter, context.partitionId(), context)
+        pythonRunnerConf,
+        sparkContext.conf).compute(projectedRowIter, context.partitionId(), context)
```

Comment:
Yea, same question.

Reply:
No, we can't normally. The tests should fail, or we need another test for this part of the Arrow code. cc @BryanCutler  (or I've misunderstood something). Whatever variables are needed from the conf need to be extracted outside of the `map` (since while RDD also has a `conf` it depends on the `sc` which is only defined on the driver.

Reply:
The conf is accessible through org.apache.spark.SparkEnv.get.conf on the executors AFAIK

Reply:
I'm updating it to `SparkEnv.get.conf`, which is what I used in other places. Thanks for catching this.